dev/com.ibm.ws.security.oauth/src/com/ibm/ws/security/oauth20/web/CoverageMapEndpointServices.java

```diff
@@ -188,12 +163,14 @@ private String validateTokenType(HttpServletRequest request, HttpServletResponse
                 new Object[] { tokenType },
                 "CWWKS1437E: Request contains unrecognized token type parameter {0}.");
         Tr.error(tc, errorMsg);
+        throw new OidcServerException(new BrowserAndServerLogMessage(tc, "OAUTH_COVERAGE_MAP_UNRECOGNIZED_TOKEN_PARAM", new Object[] { tokenType }), OIDCConstants.ERROR_INVALID_REQUEST, HttpServletResponse.SC_BAD_REQUEST);
```

Comment:
This little section needs to be reworked.
- Remove the `String errorMsg` object just like you've done everywhere else
- Ensure you're still calling `Tr.error()` to emit the error message in the server logs.
- You also have dead code underneath this line that's throwing the original `OidcServerException` that should be removed.